server/src/main/java/au/org/aodn/ogcapi/server/core/mapper/Converter.java

```diff
@@ -90,6 +90,14 @@ default <F extends StacCollectionModel> Collection getCollection(F m, String hos
                     .collect(Collectors.toList()));
         }
 
+        if (m.getSummaries() != null) {
```

Comment:
You should not have summary section in the ogc standard, please change it to use the link ok?

Reply:
What you can do is 

class ExtCollection extends Collection() {

   Map<String, ?> properties : ....... 
}

Then you do not need to touch the generated code and it will work even api upgrade unless name cashes

Reply:
got it. Thank you!